src/lib/mapper.service.ts

```diff
@@ -149,7 +149,7 @@ export class NGXMapperService {
    */
   public getCallerDetails(sourceMapsEnabled: boolean): Observable<LogPosition> {
     // parse generated file mapping from stack trace
-
+    this.logPositionRequests.clear();
```

Comment:
Wouldn't doing this make every log also do an http request for sourcemaps? I wonder why the cached local version isn't working?

Reply:
I have fixed this properly, I can submit PR later today.